lib/qs/error_handler.rb

```diff
@@ -2,12 +2,20 @@ module Qs
 
   class ErrorHandler
 
-    attr_reader :exception, :daemon_data, :job
+    attr_reader :exception, :context
     attr_reader :error_procs
```

Comment:
@jcredding I like how the error handler API is shaping up: exception, context and error procs - good stuff.  Maybe just use 1 line of readers here now?